hermes/views.py

```diff
@@ -78,19 +87,63 @@ def validate_brightness_unit(self, data):
             if row['brightness_unit'] not in brightness_units:
                 raise ValidationError(f'Unrecognized brightness unit. Accepted brightness units are {brightness_units}')
 
-
-class CandidateMessageSubSchema(Schema):
-    authors = fields.String()
-    main_data = fields.Nested(CandidateDataSchema)
-
-
-class CandidateMessageSchema(Schema):
+class MessageSchema(Schema):
     title = fields.String(required=True)
     topic = fields.String(required=True)
     event_id = fields.String()
     message_text = fields.String(required=True)
-    author = fields.String(required=True)
-    data = fields.Nested(CandidateMessageSubSchema)
+    submitter = fields.String(required=True)
+    authors = fields.String()
+    @property
+    @abstractmethod
+    def data(self):
+        pass
+
+
+class CandidateMessageSchema(MessageSchema):
+    data = fields.Nested(CandidateDataSchema)
+
+
+class PhotometryDataSchema(Schema):
+    target_name = fields.String()
+    ra = fields.String(required=True)
+    dec = fields.String(required=True)
+    date_observed = fields.String(required=True)
+    date_format = fields.String()
+    telescope = fields.String(required=True)
+    instrument = fields.String(required=True)
+    band = fields.String(required=True)
```

Comment:
Is this schema envisioned for sharing information on non-EM signals such as gravitational waves or neutrinos?  I'm just wondering how well band and brightness map in this case.  But possibly this is too general at this point.

Reply:
This is supposed to be a "photometry" schema. I'm not sure gw and neutrinos would use this.

Reply:
Fair enough; I was thinking too generally!